client/src/components/Workflow/Run/WorkflowRunName.vue

```diff
@@ -0,0 +1,71 @@
+<script setup lang="ts">
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import { type AxiosError } from "axios";
+import { BAlert } from "bootstrap-vue";
+import { onMounted, ref } from "vue";
+
+import { getWorkflowInfo } from "@/components/Workflow/workflows.services";
+
+import WorkflowInformation from "@/components/Workflow/Published/WorkflowInformation.vue";
+
+const props = defineProps({
+    model: {
+        type: Object,
+        required: true,
+    },
+});
+
+const loading = ref(true);
+const messageText = ref<string | null>(null);
+const messageVariant = ref<string | null>(null);
+const expandAnnotations = ref(true);
+const workflowInfoData = ref(null);
+
+onMounted(() => {
+    loadAnnotation();
+});
+
+async function loadAnnotation() {
+    loading.value = true;
+
+    try {
+        const workflowInfoDataPromise = getWorkflowInfo(props.model.runData.id);
+        workflowInfoData.value = await workflowInfoDataPromise;
+    } catch (e) {
+        const error = e as AxiosError<{ err_msg?: string }>;
+        messageVariant.value = "danger";
+        const message = error.response?.data && error.response.data.err_msg;
```

Comment:
You can `import { errorMessageAsString } from "@/utils/simple-error";` and use it to extract the error message.